bin/fix-dbs-parentage

```diff
@@ -51,39 +46,32 @@ from __future__ import print_function, division
 import time
 import datetime
 import argparse
+import logging
+import sys
 
 from WMCore.Services.DBS.DBS3Reader import DBS3Reader
 from dbs.apis.dbsClient import DbsApi
 
-def updateParentageInfo(datasetList, fileFix=False, insertFlag=False):
+def updateParentageInfo(datasetList, insertFlag=False, logger=logger):
 
     missingParentsFailed = {}
     start = int(time.time())
 
     if insertFlag:
         answer = input("fixing DBS parentage: Are you sure?('y'/'n') - needs ' single quote: ").lower()
```

Comment:
@amaltaro Is this one going to be used by running it manually?

Reply:
Honestly, I'm not even going to test this ad-hoc script, I believe it won't ever be used again (I'm still keeping it around though).
It has been used to recover old StepChain workflows in the system, before the cherrypy thread got created.